### ISSUE TITLE
list_image_info --per-volume for dynamic imageis blank

### ISSUE THREAD

**Issue by AnderBiguri** [UCL/STIR#698]
`list_image_info --per-volume dynamicXcat.txt`

being `dynamicXcat.txt` a multi file, prints nothing but blank.

**Comment by KrisThielemans** [comment]
yes indeed. something wrong with the option processing apparently.

I also see that `list_image_info` doesn't print time-frame info, but `get_time_frame_info` does work. The former is because of https://github.com/UCL/STIR/blob/6b47e9193f6935b4b813b07690c9a7ecc8b43961/src/buildblock/ExamInfo.cxx#L53
which should test if `get_num_times_frames()>0` of course.

It seems I did this on `release_4`, so we'll need to fix this there.
